.github/workflows/gc-raw.yml

```diff
@@ -1,84 +1,16 @@
-name: Git GC All
+name: Git GC Raw
```

Comment:
No need to change this? https://github.com/vulsio/vuls-data-db/blob/e815c2b31393ac62c6107b366b85541eaef6950f/.github/workflows/cleanup-vuls-data-db.yml#L6

Reply:
fixed: https://github.com/vulsio/vuls-data-db/compare/e815c2b31393ac62c6107b366b85541eaef6950f..e445ad0a295c92edba4c604f4e5cc394bee1b371